### DISCUSSION TITLE
Make setting Visibility by variable on Fields in Pages give a Warning

### DISCUSSION THREAD

**Discussion by SShadowS** [StefanMaron/BusinessCentral.LinterCop#204] [New Rule]
You rarely want to set the visibility on fields to something other than True or False, but instead control it by groups.
As the GUI reacts very inconsistently if you use field visibility, this should give you a warning that you are properly doing something wrong.
This will certainly help new developers not to waste a lot of time on things we cannot control.

**Comment by StefanMaron** [comment]
Maybe this can be combined into one rule together with #118

**Reply by SShadowS** [reply]
Depends on how you structure your rules.
I see two options:
- One rule for each, Editable, Enable and Visibility.
- One called Unintended/Unsupported GUI parameters on Fields and Groups on Pages
